COPYRIGHT

```diff
@@ -0,0 +1 @@
+Copyright 2022 Association of Universities for Research in Astronomy, Inc. (AURA)
```

Comment:
Are you directly paid by AURA or SLAC?  If the latter, this needs to be the SLAC copyright line ("The Board of Trustees of the Leland Stanford Junior University, through SLAC National Accelerator Laboratory").